apps/system/style/chrome/chrome.css

```diff
@@ -216,7 +216,17 @@
 .controls .urlbar {
   transform: scaleX(var(--rocketbar-scale)) translateY(0.1rem);
   transform-origin: top left;
-  transition: transform 0.2s ease-in-out;
+  transition: transform 0.2s ease-in-out, visibility 0s;
+  visibility: visible;
+}
+
+[data-manifest-name="FTU"]:not(.bar) .controls .urlbar,
+.lockScreenWindow:not(.bar) .controls .urlbar,
+.homescreen:not(.bar) .controls .urlbar,
+.globalOverlayWindow:not(.bar) .controls .urlbar,
+.appWindow.fullscreen-app .chrome:not(.dragged):not(.dragging) .controls .urlbar {
```

Comment:
This patch is only for fullscreen apps, correct? If that's the case, I would much rather have a rule only for full screen apps. One for ```.fullscreen-app .chrome.dragged, .fullscreen-app .chrome.dragging``` and one for ```.fullscreen-app .chrome```. I don't really like the idea of adding all of the extra noise in the selectors here.

Is this possible?